Cython/Distutils/build_ext.py

```diff
@@ -12,13 +12,25 @@
 
 
 class new_build_ext(_build_ext, object):
+
+    user_options = _build_ext.user_options[:]
+
+    user_options.extend([
+        ('c-build-dir=', None, "directory for generated c files"),
+    ])
```

Comment:
This can be simplified.
```suggestion
    user_options = _build_ext.user_options + [
        ('c-build-dir=', None, "directory for generated c files"),
    ]
```